docs/docusaurus/i18n/en.json

```diff
@@ -2005,8 +2006,9 @@
       "Debug": "Debug",
       "Contribute": "Contribute",
       "Technical Reference": "Technical Reference",
-      "Proposals": "Proposals",
+      "Design Documents": "Design Documents",
```

Comment:
Since the proposal process is moved to GitHub Issues, should we call this "Design Documents Archive" rather?

Reply:
Per conversation with @Neudrino , design documents are staying on Docusaurus because they are a function of the TSC. So their presence here is not archival.